uri-schemes.md

```diff
@@ -124,6 +126,88 @@ HTTP Status code: NOT 2XX, idealy the 4XX if it's user error and 5XX if it's the
 
 json object can have other properties too, but currently they are ignored by core.
 
+## **DCLOGIN** <a name="DCLOGIN"></a>
+
+|                          |                       |
+| ------------------------ | --------------------- |
+| Scheme                   | `DCLOGIN:`            |
+| Used for                 | Account setup         |
+| Related Terms\*          | Account Login QR code |
+| Available on             | draft                 |
+| Decoded by the core \*\* | draft                 |
+| Other apps using it      | none, only DeltaChat  |
+
+### Syntax
+
+```
+dclogin://user:password@host/?options
+dclogin://user:password@host?options
+dclogin:user:password@host/?options
+# example: (email: me@example.com, password: securePassword)
+dclogin://me:securePassword@example.com?v=1
+# example: (email: myself@example.com, password: 123456, insecure smtp at different server)
+dclogin://myself:123456@example.com?v=1&sh=mail.example.com&sc=3&ss=plain
+```
+
+#### Options `?options`
+
+Format: URL Query parameters also known as GET variables (`varname=value` behind the question mark, chained/delimited by `&`)
+
+The query parameters contain advanced options and a version parameter.
+All advanced options are optional except for `v`, which is the only required option at this point.
+
+| short name | full name                 | description                                         | example               |
+| ---------- | ------------------------- | --------------------------------------------------- | --------------------- |
+| `v`        | `version`                 | defines the format version, more explanation below  | `v=1`                 |
+| ---------- | ------------------------- | --------------------------------------------------- | ----------------      |
+| `ih`       | `imap_host`               | IMAP host                                           | `ih=imap.example.com` |
```

Comment:
mmh ok so actually the host can be specified in options, then splitting the email address in local part and host kinda feels unnecessary :thinking:

Reply:
it's because url spec has `username:password@host` as format

Reply:
following this "auth spec" seems not to be needed and does not really match anyway as we have to deal with potentially two different usernames and passwords.

so, trying to reconstruct the email address and username from different parts is doable, but would results in quite some "if" and unneeded complexity.

it seems much easier to have a dedicated field for the email address: `DCLOGIN:?a=foo%40bar.de&ipw=foobar` just map everything 1:1 to the existing config fields.

Reply:
I'm not sure we need to "simplify" everything that much,

maybe `DCLOGIN:email@example.com?pw=1234&v=1`, but I don't see the big benefit in specifying everything in the query part, though we could do it, minimum version (no advanced options) would then look like:
```
DCLOGIN:?a=email@example.com&p=1234&v=1
```
and when reusing the existing imap password field (though that might be more annoying to explain than an extra property):
```
DCLOGIN:?a=email@example.com&ipw=1234&v=1
```

BTW: the `v=1` is there for future proofing, that we can increase this number to force specific advanced options like oauth2 or similar in the future, if it's a number not supported by dc it will tell the user to update the app.

Reply:
I'm not really willing to change everything at this stage again, r10s is right that it might be more complicated to do it in the userinfo part (because URI encoding and stuff like that, passwords tend to contain special characters)